tests/src/rules/group-exports.js

```diff
@@ -0,0 +1,70 @@
+import { test } from '../utils'
+import { RuleTester } from 'eslint'
+import rule from 'rules/group-exports'
+
+const ruleTester = new RuleTester()
+
+ruleTester.run('group-exports', rule, {
+  valid: [
+    test({ code: 'export const test = true' }),
+    test({ code: 'export default {}\nexport const test = true' }),
+    test({ code: [
+      'const first = true',
+      'const second = true',
+      'export { first,\nsecond }',
+    ].join('\n') }),
+    test({ code: 'export default {}\n/* test */\nexport const test = true'}),
+    test({ code: 'export default {}\n// test\nexport const test = true'}),
+    test({ code: 'export const test = true\n/* test */\nexport default {}'}),
+    test({ code: 'export const test = true\n// test\nexport default {}'}),
+    test({ code: 'module.exports = {} '}),
+    test({ code: 'module.exports = { test: true,\nanother: false }' }),
+    test({ code: 'exports.test = true' }),
+  ],
+  invalid: [
+    test({
+      code: [
+        'export const test = true',
+        'export const another = true',
+      ].join('\n'),
+      errors: [
+        'Multiple named export declarations',
```

Comment:
let's have this error message contain something actionable - maybe "Multiple named export declarations; consolidate all named exports into a single statement"

Also, could this be autofixable?